src/test/unit/DelegationUnit.t.sol

```diff
@@ -3944,6 +3952,333 @@ contract DelegationManagerUnitTests_completeQueuedWithdrawal is DelegationManage
     }
 }
 
+contract DelegationManagerUnitTests_burningShares is DelegationManagerUnitTests {
+    using SingleItemArrayLib for *;
+
```

Comment:
One sanity test: withdraw all & after the withdrawal delay blocks have elapsed, nothing is burned